acceptance/gossip_peerings_test.go

```diff
@@ -45,11 +44,6 @@ func checkGossip(t *testing.T, l *localcluster.Cluster, d time.Duration,
 		case <-stopper:
 			t.Fatalf("interrupted")
 			return nil
-		case e := <-l.Events:
-			if log.V(1) {
-				log.Infof("%+v", e)
-			}
-			return fmt.Errorf("event: %+v", e)
```

Comment:
yeah, but note that the old version of this test simply throws away the events. It's strictly better now and draining the channel manually and asserting is just rewriting `AssertAndStop`. I'm not convinced this is better at this point, especially that once we actually have more than one or two of these tests it becomes way more valuable to deal with the expected events inside of the local cluster.